src/backend/api/Fusion.Resources.Api/Controllers/Requests/InternalRequestsController.cs

```diff
@@ -487,6 +492,8 @@ public async Task<ActionResult> DeleteAllocationRequest(Guid requestId)
                 {
                     if (result.Type == InternalRequestType.Allocation)
                     {
+                        or.BeRequestCreator(requestId);
```

Comment:
Should add BeRequestCreator to starting resource owner requests aswell? (can't comment on lines not changed, so adding here :) )